README.md

```diff
@@ -79,3 +90,26 @@ From here, click on the organization name and then copy the name from the pop-up
 A Bugsnag project identifier key is unique to a Bugsnag project and allows errors to be saved against it. This key can 
 be obtained by logging into Bugsnag (https://app.bugsnag.com), clicking on Projects (left side menu) and selecting the 
 required project. Once selected, the notifier API key is presented.
+
+## Testing
+
+### End-to-end (E2E)
+
+In order to run E2E tests, specific configuration and environment variables must be used.
+
+#### Auth0
+The standard Auth0 configuration can be used for the server settings. However, some additional settings must be applied
+in order for the server to get an oath token from Auth0 for creating authenticated requests. A private application
+should be created for use in Auth0 following these instructions: https://auth0.com/docs/flows/call-your-api-using-resource-owner-password-flow
+
+One special note is that the default directory must be set at the Account/tenant level. Otherwise, authorization will not
+be successful. See this StackOverflow answer for more info: https://stackoverflow.com/a/43835563/915811
+
+The special E2E client settings should be defined in `env.yml`:
+
+| Parameter | Default | Description |
+| --- | --- | --- |
+| AUTH0_CLIENT_ID | N/A | Special E2E application client ID. |
+| AUTH0_CLIENT_SECRET | N/A | Special E2E application client secret. |
```

Comment:
These new variables should appear in `env.yml.tmp`.

Reply:
I'm not sure that's the case if they're only intended for e2e.